piqueserver/networkdict.py

```diff
@@ -7,19 +7,18 @@ def get_network(cidr):
     try:
         return CACHE[cidr]
     except KeyError:
-        network = IPNetwork(cidr)
+        network = ip_network(unicode(cidr))
         CACHE[cidr] = network
         return network
 
-
 def get_cidr(network):
     # TODO: why are we accessing a protected attribute?
     #       does this work?
+    # testing for IPv4?
```

Comment:
Enet only supports IPv4. This is testing if the network is a `/32`, i.e. is a single IP address.